pipelines/validate_vset.py

```
@@ -0,0 +1,894 @@
+                )
+            )
+        if not _is_sha256(release.get("manifest_hash")):
+            errors.append(
+                VSetValidationError(
+                    "vset.actor_fields_invalid",
+                    "release.manifest_hash must be sha256:<64 hex>",
+                )
+            )
+        stamped = release.get("factory_registry_sha256")
+        if stamped is not None or require_registry_sha:
+            if stamped != pin["sha256"]:
```

Ruff (0.16.2)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)